src/ui/pages/Login/Login.tsx

```diff
@@ -12,28 +12,17 @@ const Login = (): JSX.Element => {
   return (
     <form className="flex flex-col flex-nowrap h-full login" data-testid="login-form" onSubmit={onSubmit}>
       <div className="flex flex-col items-center flex-grow p-8 login__content">
-        <Icon url={logoSVG} />
+        <Icon className="login-icon" url={logoSVG} />
 
         <div className="text-lg pt-8">
           <b>Welcome Back!</b>
         </div>
 
         <div className="text-base">To continue, please unlock your wallet</div>
 
-        <div className="py-8 w-full">
-          <Input
-            autoFocus
-            className="mb-4"
-            id="password"
-            label="Password"
-            type="password"
-            {...register("password", { required: "Password is required" })}
-          />
-        </div>
+        <PasswordInput errors={errors} isShowConfirmPassword={false} isShowInfo={false} register={register} />
```

Comment:
I'd suggest to use this approach to have components independent from any library. I mentioned about it in `PasswordInput` file and here is how to use it.

```suggestion
        <PasswordInput
            autoFocus
            className="mb-4"
            errorMessage={errors.password}
            id="password"
            isShowHints={false}
            label="Password"
            {...register("password", { required: "Password is required" })}
        />
```